shell/monitor/monitor.sh

```diff
@@ -2,13 +2,13 @@
 
 declare _monitor_pid _monitor_experiment_id _monitor_log_dir _monitor_update_es _monitor_current_test_id
 
-declare -r _monitor_eshost=aloja.bsc.es
+declare -r _monitor_eshost=https://aloja.bsc.es
 declare -r _monitor_experiment_index=metadata
 declare -r _monitor_test_index=tests
 declare -r _monitor_mapping=aloja
 
-declare -r _monitor_proxy_email=bscaloja@gmail.com
-declare -r _monitor_proxy_password=foobar
+declare -r _monitor_proxy_email="$userDashboard"
+declare -r _monitor_proxy_password="$passwordDashboard"
```

Comment:
So are these environment variables now? If so, it's more common to see ALL_CAPS variable names.

Reply:
These are supposed to come from secure/account_defaults.conf, where we already store other password and password-like stuff, and which seems to use camelCase naming conventions.

Reply:
OK, let's leave them like that then.